app/routes/preference.js

```diff
@@ -0,0 +1,18 @@
+const db = require('../data')
+
+module.exports = [ 
+  {
+    method: 'GET',
+    path: '/preference/{sbi}',
+      handler: async (request, h) => {
+        return h.response(await db.Preference.findOne({ where: { sbi: request.query.sbi } }))
```

Comment:
`Preference` should be `preference` as it needs to match the case in your model definition (app/models/preferences)

![image](https://github.com/DEFRA/ffc-sfd-permissions-api/assets/42574977/c0adc129-d55b-434a-8bf2-019ff622c720)

Reply:
Also, my bad for the example I gave you to reference.  But think it should be `request.param.sbi` not `request.query.sbi`

`param` refers to a value in the url path.  eg `/preferences/123456789` whereas `query` refers to a value in the query string. eg `/preferences?sbi=123456789`

For info, there's also `request.payload.sbi` which would take it from the body of the request when using `POST`

eg `/preferences` with a payload `{ sbi: 123456789 }`